no keystone authtoken section in cinder.conf

there is no keystone authtoken section in cinder.conf, causing cinder-api to be authorized.
ERROR keystonemiddleware.auth_token [-] HTTP connection exception: 


2015-03-04 09:52:16.566 20724 ERROR keystonemiddleware.auth_token [-] HTTP connection exception: [Errno 1] _ssl.c:510: error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol
2015-03-04 09:52:16.566 20724 WARNING keystonemiddleware.auth_token [-] Authorization failed for token
2015-03-04 09:52:16.572 20724 INFO eventlet.wsgi.server [-] 10.1.0.11 - - [04/Mar/2015 09:52:16] "GET /v1/7d24377b58ef4a77a1b37ca6293fa928/volumes/detail HTTP/1.1" 401 259 3.579256
2015-03-04 09:52:24.052 20724 INFO eventlet.wsgi.server [-] (20724) accepted ('10.1.0.11', 35536)
2015-03-04 09:52:24.068 20724 WARNING keystonemiddleware.auth_token [-] Retrying on HTTP connection exception: [Errno 1] _ssl.c:510: error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol
2015-03-04 09:52:24.585 20724 WARNING keystonemiddleware.auth_token [-] Retrying on HTTP connection exception: [Errno 1] _ssl.c:510: error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol
2015-03-04 09:52:25.604 20724 WARNING keystonemiddleware.auth_token [-] Retrying on HTTP connection exception: [Errno 1] _ssl.c:510: error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol
2015-03-04 09:52:27.622 20724 ERROR keystonemiddleware.auth_token [-] HTTP connection exception: [Errno 1] _ssl.c:510: error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol
2015-03-04 09:52:27.623 20724 WARNING keystonemiddleware.auth_token [-] Authorization failed for token
2015-03-04 09:52:27.643 20724 INFO eventlet.wsgi.server [-] 10.1.0.11 - - [04/Mar/2015 09:52:27] "GET /v1/7d24377b58ef4a77a1b37ca6293fa928/volumes/detail HTTP/1.1" 401 259 3.590356
2015-03-04 09:52:27.719 20724 INFO eventlet.wsgi.server [-] (20724) accepted ('10.1.0.11', 35542)


There should be something like this:
 
[keystone_authtoken]
...
auth_uri = http://controller:5000/v2.0
identity_uri = http://controller:35357
admin_tenant_name = service
admin_user = cinder
admin_password = CINDER_PASS

--- a/roles/cinder-controller/templates/cinder.conf
+++ b/roles/cinder-controller/templates/cinder.conf
@@ -17,5 +17,12 @@ rabbit_userid = guest
 rabbit_password = {{ RABBIT_PASS }}
 my_ip = controller
 
+[keystone_authtoken]
+auth_uri = http://controller:5000/v2.0
+identity_uri = http://controller:35357
+admin_tenant_name = service
+admin_user = cinder
+admin_password = {{ CINDER_PASS }}
+
 [database]
 connection = mysql://cinder:{{ CINDER_DBPASS }}@controller/cinder

--- a/roles/cinder-volume/templates/cinder.conf
+++ b/roles/cinder-volume/templates/cinder.conf
@@ -15,18 +15,14 @@ rabbit_host = controller
 rabbit_port = 5672
 rabbit_userid = guest
 rabbit_password = {{ RABBIT_PASS }}
-
-glance_host = controller
-
-[database]
-connection = mysql://cinder:{{CINDER_DBPASS}}@controller/cinder
+my_ip = controller
 
 [keystone_authtoken]
 auth_uri = http://controller:5000/v2.0
-# auth_host = controller
-# auth_port = 35357
-# auth_protocol = http
 identity_uri = http://controller:35357
 admin_tenant_name = service
 admin_user = cinder
 admin_password = {{ CINDER_PASS }}
+
+[database]
+connection = mysql://cinder:{{ CINDER_DBPASS }}@controller/cinder

